package.json

```diff
@@ -93,27 +91,25 @@
     "watch": "^1.0.2"
   },
   "dependencies": {
-    "@types/bluebird": "^3.5.29",
-    "@types/common-tags": "^1.8.0",
+    "@types/bluebird": "^3.5.30",
     "@types/lodash": "^4.14.149",
     "@types/memoizee": "^0.4.3",
     "@types/node": "^8.10.59",
     "abortcontroller-polyfill": "^1.4.0",
     "balena-auth": "^3.0.1",
-    "balena-device-status": "^3.2.0",
+    "balena-device-status": "^3.2.1",
     "balena-errors": "^4.2.1",
     "balena-hup-action-utils": "~3.0.0",
     "balena-pine": "^10.1.1",
     "balena-register-device": "^6.0.1",
-    "balena-request": "^10.0.4",
+    "balena-request": "^10.0.8",
     "balena-semver": "^2.1.0",
-    "balena-settings-client": "^4.0.2",
+    "balena-settings-client": "^4.0.4",
     "bluebird": "^3.7.2",
-    "common-tags": "^1.8.0",
     "lodash": "^4.17.15",
     "memoizee": "^0.4.14",
     "moment": "^2.24.0",
     "ndjson": "^1.5.0",
-    "semver": "^5.7.1"
+    "semver": "^7.1.3"
```

Comment:
Fixes a bug with intersects (which we don't use anyway) and drops old node versions we don't support: https://github.com/npm/node-semver/blob/master/CHANGELOG.md